action/protocol/staking/vote_bucket.go

```diff
@@ -260,8 +262,29 @@ func delBucket(sm protocol.StateManager, index uint64) error {
 }
 
 func getAllBuckets(sr protocol.StateReader) ([]*VoteBucket, error) {
-	// TODO
-	return nil, nil
+	// bucketKey is prefixed with const bucket = '0', all bucketKey will compare less than []byte{bucket+1}
+	maxKey := []byte{bucket + 1}
```

Comment:
It would be better to name local global const to start with underscore like `_bucket`, so it can be distinguished with func local var.

Reply:
ok, this is good one